core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -43,6 +41,9 @@ case class PSBT(
     outputMaps.size == transaction.outputs.size,
     "There must be an output map for every output in the global transaction")
 
+  import org.bitcoins.core.psbt.GlobalPSBTRecord._
+  import org.bitcoins.core.psbt.InputPSBTRecord._
```

Comment:
Why move these?

Reply:
It is required because of new inter-dependencies which cause the following if these are left top-level:
```
[error] /home/nkohen/Desktop/SuredBits/bitcoin-s-core/core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala:11:31: encountered unrecoverable cycle resolving import.
[error] Note: this is often due in part to a class depending on a definition nested within its companion.
```
essentially these must be moved to avoid circular dependencies